data/filter/Types.ts

```diff
@@ -48,4 +48,13 @@ export type FilterLike = Filter |
     FieldFilterSpec |
     FunctionFilterSpec |
     FilterTestFn |
-    FilterLike[];
\ No newline at end of file
+    FilterLike[];
+
+/**
```

Comment:
Can use Omit<FilterLike, FunctionFilter> for clarity.  Also -- do really need this type? Can we use it inline in FilterChooserCode?

Reply:
We also have to exclude `FilterTestFn` and `FilterLike[]` and add `FilterChooserFilterLike[]`, so not sure if the Omit syntax is any clearer in the end, but good to know about it! I'll move it to FilterChooserModel as a local type, agree it doesn't need to be here.